lib/components/narrative/default/default-itinerary.js

```diff
@@ -2,50 +2,179 @@ import coreUtils from '@opentripplanner/core-utils'
+            paddingLeft: '4px',
+            paddingBottom: '4px'
+          }}>
+            <LegIcon leg={leg} />
+          </div>
+        </span>
+      )
+    }
+  }
+]
+
 export default class DefaultItinerary extends NarrativeItinerary {
+  _onMouseOver = () => {
+    const {index, setVisibleItinerary} = this.props
+    if (typeof setVisibleItinerary === 'function') setVisibleItinerary({index})
+  }
+
+  _onMouseOut = () => {
+    const {setVisibleItinerary} = this.props
+    if (typeof setVisibleItinerary === 'function') setVisibleItinerary({index: null})
+  }
+
+  _isSortingOnAttribute = (attribute) => {
+    const {sort} = this.props
+    if (sort && sort.type) {
+      const type = sort.type.toLowerCase()
+      return attribute.id.toLowerCase() === type ||
+        (attribute.alias && attribute.alias.toLowerCase() === type)
+    }
+    return false
+  }
+
   render () {
     const {
       active,
-      activeLeg,
-      activeStep,
       expanded,
-      index,
       itinerary,
       LegIcon,
-      setActiveLeg,
-      setActiveStep
+      timeFormat
     } = this.props
+    const timeOptions = {
+      format: timeFormat,
+      offset: coreUtils.itinerary.getTimeZoneOffset(itinerary)
+    }
     return (
-      <div className={`option default-itin${active ? ' active' : ''}`}>
+      <div
+        className={`option default-itin${active ? ' active' : ''}${expanded ? ' expanded' : ''}`}
+        role='presentation'
+        // FIXME: Move style to css
+        style={{
+          borderLeft: active && !expanded ? '4px teal solid' : undefined,
+          backgroundColor: expanded ? 'white' : undefined
+        }}
+        onBlur={this._onMouseOut}
+        onFocus={this._onMouseOver}
+        onMouseOut={this._onMouseOut}
+        onMouseOver={this._onMouseOver}
+      >
         <button
           className='header'
+          // _onHeaderClick comes from super component (NarrativeItinerary).
           onClick={this._onHeaderClick}
         >
-          <span className='title'>Itinerary {index + 1}</span>{' '}
-          <span className='duration pull-right'>{formatDuration(itinerary.duration)}</span>{' '}
-          <span className='arrivalTime'>{formatTime(itinerary.startTime)}—{formatTime(itinerary.endTime)}</span>
+          <div
+            className='title'>
+            {getItineraryDescription(itinerary)}
+          </div>
+          <ul className='list-unstyled itinerary-attributes'>
+            {ITINERARY_ATTRIBUTES
+              .sort((a, b) => {
+                const aSelected = this._isSortingOnAttribute(a)
+                const bSelected = this._isSortingOnAttribute(b)
+                if (aSelected) return -1
+                if (bSelected) return 1
+                else return a.order - b.order
+              })
+              .map(attribute => {
+                const isSelected = this._isSortingOnAttribute(attribute)
+                const options = attribute.id === 'arrivalTime' ? timeOptions : {}
```

Comment:
There is an alignment issue for late morning/evening times:

![image](https://user-images.githubusercontent.com/56846598/88728776-7157b500-d100-11ea-8d0f-c473ebc0e936.png)

Reply:
This block is currently set to float right. Not sure the best way to approach a fix for this. Any thoughts?

Reply:
Use a `min-width` for these blocks?

Reply:
No, doesn't seem to work. This PR is getting stale, so I'm going to move this to an issue.

Reply:
I posted an alternate layout idea in #211.